tests/konnector_structure.coffee

```diff
@@ -10,7 +10,9 @@ describe 'Check all konnectors', ->
     listOfKonnectors = fs.readdirSync konnectorsDirectory
     listOfLocales = fs.readdirSync localesDirectory
 
-    listOfKonnectors.forEach (filename) ->
+    # Remove hidden files (a.k.a. thoses starting with '.')
+    listOfKonnectors.filter (filename) -> filename.indexOf('.') != 0
```

Comment:
Are you sure you want to compare to 0 ? By default if it does not find the string it returns -1.
Besides, please prefer `!==` to `!=`.